File: sources/person.rs
use std::collections::HashMap;
use std::default::Default;

impl Person {
    fn new() -> Self {
        Self {
            name: "Ethan Lerner",
            aliases: vec!["miapolis"],
            socials: HashMap::from([
                ("GitHub", "https://github.com/miapolis"),
                ("Discord", "miapolis#????"),
                ("Spotify", "https://l.lerners.io/spotify"),
                ("Steam", "https://l.lerners.io/steam"),
            ]),
        }
    }
}

File: sources/person.py
from dataclasses import dataclass, field
from typing import List, Dict

@dataclass
class Person:
    name: str = "Ethan Lerner"
    aliases: List[str] = field(default_factory=lambda: ["miapolis"])
    socials: Dict[str, str] = field(default_factory=lambda: {
        "Email": "ethan@lerners.io",
        "GitHub": "https://github.com/miapolis",
        "Discord": "miapolis#????",
        "Spotify": "https://lerners.io/spotify",
        "Steam": "https://lerners.io/steam",
    })
